src/serviceregistry/5_0_0/sysd/serviceregistry_sysd.tex

```diff
@@ -0,0 +1,228 @@
+
+\subsection{System functionalities and properties}
+\label{sec:properties}
+
+\subsubsection {Functional properties of the system}
+Service Registry solves the following needs to fulfill the requirements of service/system/device discovery.
+
+\begin{itemize}
+    \item Enables the application and other core/support systems to register themselves in order to being part of the Local Cloud.
+    \item Enables the application and other core/support systems to unregister themselves from the Local Cloud.
+    \item Enables the application and other core/support system to register the device on which the system is running.
+    \item Enables the application and other core/support system to unregister the device on which the system is running (only if no other systems belongs to that device).
+    \item Enables the application and other core/support systems to publish their services within the Local Cloud.
+    \item Enables the application and other core/support systems to revoke their services from the Local Cloud.
+    \item Makes the offered services/systems/devices queryable for other systems within the Local Cloud.
+\end{itemize}
+
+\subsubsection {Non functional properties of the system}
+If an Authentication system is present in the Local Cloud, the Service Registry will use its service(s) to verify a requester system before responding to its request. If X.509 authentication policy is used instead of a dedicated Authentication system, the Service Registry will verify the provided certificate before any response.
+
+\subsubsection {Data stored by the system}
+In order to achieve the mentioned functionalities, Service Registry is capable to store the following information set:
+
+\begin{itemize}
+    \item \textbf{Device}: name, various types (e.g. MAC address) of addresses of the device and custom meta data.
+    \item \textbf{System}: name, version, various types (e.g. IP address) of addresses of the system, an optional reference to the device on which the system is running, custom meta data. 
+    \item \textbf{Service definition}: name.
+    \item \textbf{Interface}: name, communication protocol, optional validation data.
+    \item \textbf{Service instance}: name, service definition and system reference, version, expiration date and time, instance level custom meta data, interface reference, security policy, and interface-related data (e. g. IP addresses, port, base path, content type, input and output data models).
+\end{itemize}
+
+\subsection{Important Delimitations}
+\label{sec:delimitations}
+
+\begin{itemize}
+    \item Names (device name, system name, service definition name, service instance name, interface name) must be unique in the Local Cloud.
+    \item If the Local Cloud does not contain an Authentication system or not using X.509 certificates, there is no way for the Service Registry to verify the requester system. In that case, the Service Registry will consider the authentication data comes from the requester as valid.
```

Comment:
- "or not using X.509 certificates" -> "or **does** not **use** X.509 certificates"
- "authentication data comes from the requester" -> "authentication data **that** comes from the requester"